test/functional/Valhalla/build.xml

```diff
@@ -62,6 +62,7 @@
 			<!-- <src path="${src_lw5}"/> -->
 			<src path="${transformerListener}" />
 			<compilerarg line='--add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/jdk.internal.value=ALL-UNNAMED' />
+			<compilerarg line='--enable-preview -source 22'/>
```

Comment:
lw5 seems to introduce an new option `-XDenableNullRestrictedTypes` to enable null restricted type.

Reply:
You are correct. I'm compiling these tests with lw5 commit dc715159feda32945d5c2351ede9c6be04a64d6f which is before aconst_init/vnew/withfield are removed and also before that command line option is added. `-XDenableNullRestrictedTypes` will be added once its compiling at the head of lw5.